drizzlepac/hlautils/photometry_graphics.py

```diff
@@ -0,0 +1,263 @@
+# This routine is designed specifically to support plotting the
+# SVM data quality output.  As such, it expects the input Pandas
+# Dataframe to have a particular structure and the data to contain
+# specific measurements.
+#
+# This is NOT a general plotting utility and is a prototype.
+# To use:
+# 1) Start a Python session
+# 2) import photometry_graphics as pg
+# 3) You need to provide a fully qualified pathname to the directory
+#    above all of the output directories which contain the photometry
+#    JSON output files.
+# 4) In the Python session, invoke the harvester
+#    >>> generate_photometry_df(input_directory, output_csv_filename='svm_photometry_stats.csv')
+#        This will generate an output CSV file where a Pandas
+#        dataframe is stored.
+# 5) Once the CSV file is generated, invoke the graphics routine
+#    >>> pg.photometry_graphics_driver(CSV_filename)
+#        This will generate a Bokeh plot to the browsers, as
+#        well as generate an HTML file.
+#
+# SVM Photometry file has the following data values where
+# differences are (POINT - SEGMENT).
+# Delta_MagAp1-Mean Difference: -0.04963999999999993
+# Delta_MagAp1-Standard Deviation: 0.0869423778526139
+# Delta_MagAp1-Median Difference: -0.0259999999999998
+# Delta_MagAp2-Mean Difference: 0.009571428571428555
+# Delta_MagAp2-Standard Deviation: 0.02693804431083513
+# Delta_MagAp2-Median Difference: 0.006000000000000227
+# ISSUES:
+# Too many columns from header in JSON cluttering up dataframe.
+# Hierachical information in JSON not tightly coupled to values once in dataframe.
+# To use hover, MUST have all data columns coming from ColumnDataSource.
+# ColumnDataSource needs to use columns names, so use succinct names.
+# Think about information useful for hover.
+
+import array
+from bokeh.layouts import row
+from bokeh.plotting import figure, output_file, show
+from bokeh.models import ColumnDataSource, Label
+import csv
+import glob
+import json
+import os
+import pandas as pd
+
+
+def compute_global_stats(value_array):
```

Comment:
Style comment: 'compute_global_stats' implies the computation of multiple statistics terms, not just the mean.  This is confusing unless it was intended to serve as place-holder for computing additional stats later (std, median, ...).  No changes needed here, but it would be useful to keep this in mind in the final implementation.

Reply:
It is meant to be a placeholder to compute multiple stats.  The function should have a better description or a better name depending upon the circumstances!